test.js

```diff
@@ -95,6 +95,23 @@ test('.has()', t => {
 	t.false(t.context.config.has('missing'));
 });
 
+test('.reset()', t => {
+	t.context.configWithSchema = new Conf({cwd: tempy.directory(), schema: {foo: {default: 42}, bar: {default: 99}}});
+	t.context.configWithDefaults = new Conf({cwd: tempy.directory(), defaults: {foo: 42, bar: 99}});
```

Comment:
Put the object items on separate lines. It's hard to read when everything is on one line.